cmd/policy/policy.go

```diff
@@ -0,0 +1,69 @@
+package policy
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/spf13/cobra"
+
+	"github.com/CircleCI-Public/circleci-cli/api/policy"
+
+	"github.com/CircleCI-Public/circleci-cli/settings"
+)
+
+func NewCommand(config *settings.Config, preRunE validator) *cobra.Command {
+	var policyClient ClientInterface
+
+	initClient := func(cmd *cobra.Command, args []string) (e error) {
+		if policyClient, e = policy.NewClient(*config); e != nil {
+			return e
+		}
+		return preRunE(cmd, args)
+	}
+
+	command := &cobra.Command{
+		Use: "policy",
+		Short: "Policies ensures security of build configs via security policy management framework. " +
+			"This group of commands allows the management of polices to be verified against build configs.",
+	}
+
+	listPoliciesCommand := &cobra.Command{
+		Short:   "List all policies",
+		Use:     "list <ownerID> [<active>]",
+		PreRunE: initClient,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return listPolicies(policyClient, args)
+		},
+		Args:        cobra.RangeArgs(1, 2),
```

Comment:
using flags for owner and active filter would be much nicer than positional args.